src/pretix/base/settings.py

```diff
@@ -546,8 +546,8 @@ def unserialize(cls, s):
         'form_kwargs': dict(
             label=_("Ask for VAT ID"),
             help_text=format_lazy(
-                _("Only works if an invoice address is asked for. VAT ID is never required and only requested from "
-                  "business customers in the following countries: {countries}"),
+                "Only works if an invoice address is asked for. VAT ID is never required and only requested from "
+                "business customers in the following countries: {countries}",
```

Comment:
I believe this is wrong. If you take the ``_`` away, `makemessages` will no longer find this string as a translatable one *at all*, so it will no longer be included in .po files if you run makemessages and will no longer get translated, or am I missing something I don't know about gettext?

Reply:
Hm, django-docs recommend using format_lazy. When using _() before, then when having a default locale other en, this will break.

Reply:
Fixed.